content/_releases/24/23.md

```diff
@@ -0,0 +1,95 @@
+category: product-updates
+releaseDate: 2024-06-06
+redirect_from:
+  - /releases/24.23.html
+---
+
+## New Features
+### Enhanced flow name visibility in filter dropdowns
+Improved the flow name visibility in the filter dropdowns on the `Executions`, `Containers`, and `Logs` pages. 
+Now, when hovering over flow names in the filter dropdowns, users can view the full flow name if it's too long. This update facilitates better understanding and navigation within the platform, making it easier for users to identify and select specific flows for their tasks.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2423/full_flow_name.gif" title="Flow Name" %}
+
+### Bulk flow Export feature
+Experience increased productivity with the new bulk flow export capability on the platform. The new dropdown menu `Exports` now offers users two convenient options: `Export Flow(s)` and `Export to a Recipe`.
+Selecting `Export Flow(s)`, users can choose one or multiple flows and export them simultaneously to any available contract and workspace as flow drafts. This feature empowers users to efficiently manage and export multiple flows with ease, optimizing their workflow and boosting productivity.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2423/many_flows.gif" title="Export Many Flows" %}
+
+### Improved data visualization in JSONataView with `label` field display
+Enhanced data representation in the JSONataView viewclass with the introduction of `label` field display. Users can now easily access and view the contents of the label field within JSONataView, as described in the `component.json` file. This enhancement allows for clearer data visualization and interpretation within the viewclass, and provides users with enabling better understanding and analysis of the label field content within the JSONataView viewclass.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2423/Label.png" title="Label" %}
+
+### Unsubscribing from flow errors via API
+With the new API endpoint `https://api.elastic.io/docs/v2#/flows/delete_flows_subscription`, users can now unsubscribe from flow errors across all their flows or specifically within a specific workspace. 
+This feature empowers integrators to efficiently manage error notifications, providing flexibility and control over error subscriptions for enhanced workflow efficiency.
+
+### `Default` label display for Router component branches
+Users can now easily view the `Default` label associated with a branch of the router component directly on the flow queue page. 
+This update provides users with quick access to essential information, facilitating better understanding and management of the routing logic within the flow queue.
+
+### Updated status code `429` for exceeded Webhook requests limit
+When the quota for Webhook requests is exceeded, the Webhook URL now responds with a `429 Too Many Requests` status code. 
+
+### Symbols verification for Webhook Service Credentials
```

Comment:
Webhook component credentials (we didn't touch the servise, just validation on UI )